Docs/source/usage/parameters.rst

```diff
@@ -193,9 +193,14 @@ Domain Boundary Conditions
     Options are:
 
     * ``Periodic``: This option can be used to set periodic domain boundaries. Note that if the fields for lo in a certain dimension are set to periodic, then the corresponding upper boundary must also be set to periodic. If particle boundaries are not specified in the input file, then particles boundaries by default will be set to periodic. If particles boundaries are specified, then they must be set to periodic corresponding to the periodic field boundaries.
+
     * ``pml`` (default): This option can be used to add Perfectly Matched Layers (PML) around the simulation domain. It will override the user-defined value provided for ``warpx.do_pml``. See the :ref:`PML theory section <theory-bc>` for more details.
     Additional pml algorithms can be explored using the parameters ``warpx.do_pml_in_domain``, ``warpx.do_particles_in_pml``, and ``warpx.do_pml_j_damping``.
 
+    * ``pec``: This option can be used to set a Perfect Electric Conductor (PEC) boundary, which is one of the reflective domain boundaries. At PEC, the tangential electric field and the normal magnetic field are set to 0. This boundary can be used to model a dielectric or metallic surface. Note that PEC boundary is invalid at `r=0` for the RZ solver. Please use ``none`` option.
+
+    * ``none``: No boundary condition is applied to the fields. This option can be used for the RZ-solver at `r=0`.
```

Comment:
@dpgrote @RemiLehe Is this statement correct for RZ. Please also see the corresponding recommendation under "pec" for RZ at r=0.

Reply:
This is good, though I would make it more definite, saying "This option must be used for the RZ-solver at 'r=0'.".